src/matrixrtc/MembershipManager.ts

```diff
@@ -325,7 +312,7 @@ export class MembershipManager
         if (userId === null) throw Error("Missing userId in client");
         if (deviceId === null) throw Error("Missing deviceId in client");
         this.deviceId = deviceId;
-        this.stateKey = this.makeMembershipStateKey(userId, deviceId);
+        this.memberId = this.makeMembershipStateKey(userId, deviceId);
```

Comment:
Is this meant to be a state key or a member ID? It looks like this is used as both, but I was under the impression that member IDs were supposed to be unique to each period of connection to the slot.